libs/wingsdk/src/cloud/function.md

```diff
@@ -56,12 +56,12 @@ new cloud.Function(inflight () => {
 
 ## Function container reuse
 
-Most cloud providers will opportunistically reuse the function's container in additional invocations. It is possible
-to leverage this behavior to cache objects across function executions using `inflight new` and inflight fields.
+Most cloud providers will opportunistically reuse the function's container in additional invocations.
+It is possible to leverage this behavior to cache objects across function executions using `inflight new` and inflight fields.
```

Comment:
imo it feels like a bad practice to encourage this way of thinking unless `cloud.Function` is intentionally designed as a "Stateful Function`.

Reply:
Hmm, yeah that's a good point. Maybe this deserves some more discussion (@ShaiBer also raised some questions about whether this makes sense to have as a functional concern in the cloud.Function API)